README.md

```diff
@@ -13,9 +13,30 @@ Helium is a frontend web development framework for building highly contextual, d
 Helium utilizes a modern technology stack that includes React, GraphQL, and Tailwind CSS to provide an exceptional developer experience. The out-of-the-box UI components make getting started easy and with GraphQL getting to the data you need is intuitive and fast.
 
 ## Getting Started
-#### Requirements:
+### Requirements:
 `npm` and `node` are pinned to specific versions using [Volta](https://volta.sh/), so you will want to [install it](https://docs.volta.sh/guide/getting-started) before anything else.
 
+### Installing Locally
+To install and run this Helium repo locally, clone the repo and run the following commands:
```

Comment:
looks good; thinking we may want to include the commands for cloning the repo just in case?